examples/config-multiple.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.1.0-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot",
```

Comment:
shouldn't this simply be "/" in the subpaths section? The subpath is obviously the longest prefix, right? This will allow us to operationally move subpaths into different volumes on a running server in the future.

Reply:
I didn't understand your point, could you please explain in detail ?

Reply:
Ignore it. longest prefix has issues.